Unit test failing on ci/cd pipeline test_static_web_ui tests/unit/test_core/test_api_server.py:92

The following unit test is failing on the CI/CD pipeline daily execution:

```
=================================== FAILURES ===================================
____________________ TestAPIServer.test_static_web_ui[True] ____________________

self = <tests.unit.test_core.test_api_server.TestAPIServer object at 0x7f6df94faf10>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f6df8645690>
file_exist = True

    @pytest.mark.parametrize("file_exist", [True, False])
    async def test_static_web_ui(self, monkeypatch, file_exist):
        """Test static_web_ui method to success case."""
        monkeypatch.setattr("os.path.exists", lambda x: file_exist)
        monkeypatch.setattr("kytos.core.api_server.FileResponse",
                            lambda x: JSONResponse({}))
    
        endpoint = f"/ui/kytos/napp/{self.api_server.napps_dir}"
        client = AsyncClient(app=self.app, base_url="http://127.0.0.1")
>       response = await client.get(endpoint)

tests/unit/test_core/test_api_server.py:92: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
..
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <kytos.core.api_server.APIServer object at 0x7f6df869af50>
request = <starlette.requests.Request object at 0x7f6df86f29d0>

    def static_web_ui(self,
                      request: Request) -> Union[FileResponse, JSONResponse]:
        """Serve static files from installed napps."""
        username = request.path_params["username"]
        napp_name = request.path_params["napp_name"]
        filename = request.path_params["filename"]
        path = f"{self.napps_dir}/{username}/{napp_name}/ui/{filename}"
        if os.path.exists(path):
            headers = {
                "Cache-Control": "no-cache, no-store, must-revalidate",
                "Pragma": "no-cache",
                "Expires": "0"
            }
>           return FileResponse(path, headers=headers)
E           TypeError: TestAPIServer.test_static_web_ui.<locals>.<lambda>() got an unexpected keyword argument 'headers'

kytos/core/api_server.py:188: TypeError
------------------------------ Captured log setup ------------------------------

```

--- a/tests/unit/test_core/test_api_server.py
+++ b/tests/unit/test_core/test_api_server.py
@@ -85,7 +85,7 @@ class TestAPIServer:
         """Test static_web_ui method to success case."""
         monkeypatch.setattr("os.path.exists", lambda x: file_exist)
         monkeypatch.setattr("kytos.core.api_server.FileResponse",
-                            lambda x: JSONResponse({}))
+                            lambda x, headers: JSONResponse({}))
 
         endpoint = f"/ui/kytos/napp/{self.api_server.napps_dir}"
         client = AsyncClient(app=self.app, base_url="http://127.0.0.1")

